apps/src/gamelab/GameLab.js

```diff
@@ -177,6 +216,34 @@ GameLab.prototype.init = function (config) {
   });
 };
 
+function quote(str) {
+  return '"' + str + '"';
+}
+
+/**
+ * Returns a list of options (optionally filtered by type) for code-mode
+ * asset dropdowns.
+ */
+GameLab.prototype.getAssetDropdown = function (typeFilter) {
```

Comment:
Again, this function is (for now) exactly duplicated from applab.js.  Let's extract it.